examples/c-tutorial-2-streams.c

```diff
@@ -16,7 +16,7 @@
 
 /* Function calls to librealsense may raise errors of type rs_error */
 rs_error * e = 0;
-void check_error()
+void       check_error()
```

Comment:
Ok, let me fix it. It's because clang-format tries to align to above line.

Note: some librealsense code has following alignment.
```
static rs_context*                              acquire_instance();
static void                                         release_instance();
static int                                            ref_count;
static std::mutex                               instance_lock;
static rs_context*                             instance;
```

It'll be changed to
```
static rs_context* acquire_instance();
static void release_instance();
static int ref_count;
static std::mutex instance_lock;
static rs_context* instance;
```